integrations/malware_tools_analyzers/Dockerfile

```diff
@@ -159,6 +164,14 @@ RUN mkdir -m 0770 -p /var/lib/fangfrisch \
     && touch /var/log/cron.log \
     && chmod 0644 /etc/cron.d/crontab /var/log/cron.log
 
+# Install Guarddog
+# Not working with virutalenv
+WORKDIR ${PROJECT_PATH}/guarddog
+
+RUN pip3 install --no-cache-dir --upgrade pip \
+    && pip3 install --no-cache-dir guarddog
+    
+
```

Comment:
the ideal thing would be to use its own python virtual environment like we did for all of the other integrations in this image to avoid conflicts. You can copy how we did that for the others

Reply:
(you can add a new file in the `requirements` folder with that guardog dependency and pinning a version for reproducibility)